internal/pkg/build/sources/conveyorPacker_library.go

```diff
@@ -70,14 +81,47 @@ func (cp *LibraryConveyorPacker) Get(b *types.Bundle) (err error) {
 		}
 	}
 
+	if !cp.AllowUnauthenticated {
```

Comment:
this entire branch should be a different function, say `func (cp *LibraryConveyorPacker) authenticate(...) error`